index.html

```diff
@@ -90,4 +90,13 @@ <h3 class="post-subtitle">
      </article>
     {% endfor %}
   </div>
+
+  <!-- 2020 fundraiser sponsor logos -->
+  <div id="fundraiser">
+	  <h2 style="font-size:42px;">Supporters</h2>
+	  <p>Bitcoin ABC relies on sponsorships to fund all its work, including devops, maintenance, backports, development, release management, technical support and more. Become a supporter today at <a target=_fund" href="https://fund.bitcoinabc.org/">fund.bitcoinabc.org</a>.</p>
```

Comment:
target is missing quotes

Reply:
checked all targets, seem to be good now.